tests/test_dates.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+import pytest
+
+# used for reproducible relative date testing
+FIXED_DATE = datetime.date(2020, 3, 25)
+
+@pytest.fixture
+def patch_datetime_today(monkeypatch):
+    class FixedDate(datetime.date):
+        @classmethod
+        def today(cls):
+            return FIXED_DATE
+    monkeypatch.setattr(datetime, 'date', FixedDate)
```

Comment:
This seems to be doing the same thing as `freeze_time` used in [test_date_disambiguator](https://github.com/nextstrain/augur/blob/master/tests/util_support/test_date_disambiguator.py)?

Reply:
Didn't realize `freezegun` was already used here. Thanks for the pointer!

Updated to use this, and have consistent styling for `pytest.mark.parametrize`: [diff](https://github.com/nextstrain/augur/compare/8758ad25..a4620431)